### DISCUSSION TITLE
大通缩已经实质性开启半年了，各位保重。

### DISCUSSION THREAD

**Discussion by renjianbingqi** [WeNeedHome/SummaryOfLoanSuspension#931] [General]
1：M0 缩水1万亿，M2金融空转100万亿左右
2：财政缩水   体制内工资缩水
3：外贸订单缩水更严重，下半年几乎订单膝盖斩 
4：人口
5：信心

保存好你们最后的现金，迎接这波5年起步的大通缩。

**Comment by Accelerator6666** [comment]
现在最主要考虑的就是把所有金融理财类产品全都赎回，提前结清债务（没有债务时最理想的状态），取消一切非必要开销，尽量避免大笔现金支出。保证自己现金安全性。有机会兑一些外币现金避险。

**Reply by CarberryChai** [reply]
> 现在最主要考虑的就是把所有金融理财类产品全都赎回，提前结清债务（没有债务时最理想的状态），取消一切非必要开销，尽量避免大笔现金支出。保证自己现金安全性。有机会兑一些外币现金避险。

余额宝安全嘛？？

**Reply by Accelerator6666** [reply]
> 
余额宝也是金融产品哦

**Reply by imhappyok** [reply]
老百姓要都按你这么玩，经济怎么流通，估计大部分人要失业降薪了。。。

**Reply by Accelerator6666** [reply]
> 老百姓要都按你这么玩，经济怎么流通，估计大部分人要失业降薪了。。。

老百姓的衣食行不都是经济流通。

**Reply by shellfeng** [reply]
网商银行安全不？

**Reply by jslygljy** [reply]
信心胜过黄金

**Comment by blisee** [comment]
百年未有之大变局！

**Reply by miaodi** [reply]
大便菊

**Reply by jim486** [reply]
数字货币会加快推出，偸着乐系列

**Reply by terrywangt** [reply]
千年未有之大变局！

**Comment by yuyachen** [comment]
来炒股吧，新手1万块以下玩玩，最近新能源，光伏，机器人涨的不错，每天几十家到上百家涨停，看你有没有这个本事，挂单买卖都是明牌，相对来说公平点。

**Reply by xiazhifengtest** [reply]
大A股吊打韭菜

**Reply by le840923533** [reply]
加速GG

**Reply by YamaXanadu830** [reply]
不能做空的不炒

**Reply by shawn-yee** [reply]
一万块不重要，时间和精力和感情都很重要。

**Comment by stopium** [comment]
外币不一定安全啊，这种时候，价格合理的固定资产比较安全吧

**Reply by Accelerator6666** [reply]
关键就是现在没啥价格合理的固定资产。。。

**Comment by pinerose99** [comment]
> 1：M0 缩水1万亿，M2金融空转100万亿左右 2：财政缩水 体制内工资缩水 3：外贸订单缩水更严重，下半年几乎订单膝盖斩 4：人口 5：信心
> 
> 保存好你们最后的现金，迎接这波5年起步的大通缩。

请贴出央行财务报表的数据。

**Reply by renjianbingqi** [reply]
> > 1：M0 缩水1万亿，M2金融空转100万亿左右 2：财政缩水 体制内工资缩水 3：外贸订单缩水更严重，下半年几乎订单膝盖斩 4：人口 5：信心
> > 保存好你们最后的现金，迎接这波5年起步的大通缩。
> 
> 请贴出央行财务报表的数据。

人民银行官网，自己去找，每月都有。

**Reply by carlislelee** [reply]
楼主说的什么时候和什么时候比缩水1w亿？
![image](https://user-images.githubusercontent.com/6078783/182192196-d50a27ea-b23b-4284-931a-d0e14cd0db2e.png)

**Reply by zongmekeneng** [reply]
如上图所示
流通中货比（M0）   2022/01 -- 2022-06   缩水 1W亿

**Comment by Elneanor** [comment]
果然是现金为王.

**Comment by shawn-yee** [comment]
未来五年，大概率是真的了。
